src/cgame/cg_main.cpp

```diff
@@ -1201,6 +1201,8 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 {
 	const char *s;
 
+	Rocket_SetActiveContext( KEYCATCH_UI );
```

Comment:
The new version using Rocket_SetActiveContext doesn't work right. The call here at the beginning of CG_Init doesn't actually end up calling trap_SetMouseMode.